packages/wallets/src/metamask/metamask.constants.ts

```diff
@@ -7,4 +7,5 @@ export const METAMASK_COMMON_CONFIG: CommonWalletConfig = {
   CONNECT_BUTTON_NAME: 'Metamask',
   SIMPLE_CONNECT: false,
   EXTENSION_START_PATH: '/home.html',
+  TESTNET_NETWORK: 'Goerli',
```

Comment:
not sure if it should be here, you can put it directly in tests

Reply:
I suppose it can describe wallet from "Does wallet support testnet networks?" point of view

Reply:
Decided to remove it for now